djangocms_text_ckeditor/forms.py

```diff
@@ -83,6 +83,8 @@ def get_child_plugins(self):
             excluded_plugins = []
 
         queryset = self.text_plugin.get_descendants()
+        if not queryset.exists():
+            queryset = CMSPlugin.objects.all()
```

Comment:
If the `text_plugin` contains no descendants the queryset will be empty. `DeleteOnCancelForm` will thus be invalid when it is checked. So instead of getting a 200 error you'll get a 400 Bad Request.